src/ServiceControl.AcceptanceTests/Recoverability/ExternalIntegration/When_a_failed_message_is_archived.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace ServiceControl.AcceptanceTests.Recoverability.ExternalIntegration
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NUnit.Framework;
+    using Contracts;
+    using ServiceControl.MessageFailures;
+    using Newtonsoft.Json;
+    using TestSupport.EndpointTemplates;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    class When_a_failed_message_is_archived : When_a_message_failed
```

Comment:
Is it possible to rename the base class to not contain `When_`? It know this sounds super nitpicky but for my brain a `When_` is always a concrete scenario but here it seems `When_a_message_failed` is just a convenience base class that provides infrastructure to avoid having to repeat things for external integration. If the fact that all those events are based upon message failure how about calling it something like 

- `MessageFailedAcceptanceTests` or
- `MessageFailedIntegrationAcceptanceTests`